docs/index.rst

```diff
@@ -11,6 +11,30 @@ Everything else is on
 `Github Wiki <https://github.com/Ulauncher/Ulauncher/wiki/>`_.
 
 
+Launch-Targets
+==============
+
+Additionally to launching Applications, Ulauncher can launch the following targets:
+
+Desktop-Entries
+---------------
+
+Ulauncher indexes desktop-entries in ``~/.local/share/applications`` the same as applications.
+Using desktop-entries, launchers that require no query (no space or arguments other than the name/keyword) can be created easily.
+
+
+Shortcuts
+---------
+
+Shortcuts configure Web-URLs using a Query-String or Script, to be opened in the default browser.
+
+
+Extensions
```

Comment:
Extensions are described in detail on the main Ulauncher site. The docs you are updating are for developers who by the time of writing an extensions should have read what an extension is.

Reply:
I think that's fine